CHANGELOG.md

```diff
@@ -89,6 +89,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   reversing colors rather than underlining, you can set
   `colors."diff token"={ underline = false, reverse = true }` in your config.
 
+* `jj status` and output showing the new working copy commit now includes `@`
+  and `@-` to help newcomers learn the symbols.
```

Comment:
nit: I don't think we need to mention it in the release notes. Users don't need to act on it, and it probably doesn't reduce surprises much either (it should be pretty obvious what the symbols mean to any existing users).